fixieai/agents-ts/package.json

```diff
@@ -27,19 +30,34 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@tsconfig/node18": "^1.0.1",
+    "@types/bunyan-format": "^0.2.5",
+    "@types/jest": "^29.5.0",
+    "@types/js-yaml": "^4.0.5",
+    "@types/lodash": "^4.14.192",
+    "@types/yargs": "^17.0.24",
     "@typescript-eslint/eslint-plugin": "^5.57.1",
     "@typescript-eslint/parser": "^5.57.1",
     "dprint": "^0.35.3",
     "eslint": "^8.37.0",
     "eslint-config-nth": "^2.0.1",
+    "jest": "^29.5.0",
+    "nock": "^13.3.0",
     "nodemon": "^2.0.21",
-    "ts-node": "^10.9.1",
+    "ts-jest": "^29.1.0",
     "ts-node-dev": "^2.0.0",
     "typescript": "^5.0.3"
   },
   "dependencies": {
     "@types/express": "^4.17.17",
-    "express": "^4.18.2"
+    "bunyan": "^1.8.15",
+    "bunyan-format": "^0.2.1",
+    "bunyan-middleware": "^1.0.2",
+    "express": "^4.18.2",
+    "got": "^11.8.6",
```

Comment:
got@12 uses ESM in a way that's really hard for us to use, so we can just use got@11 instead.